### PR TITLE
Normal but folded composer output

### PR DESCRIPTION
**Pull request by greg0ire**

### CODE REVIEW

**Comment by greg0ire** [comment]
Any idea how to test this @OskarStark ?

**Comment by greg0ire** [comment]
maybe if you push this in a separate branch on your repo?

**Comment by greg0ire** [comment]
I found how to test! Don't merge, it does not work yet, but it looks promising: for now I only seem to be able to get the warning, not sure why.

EDIT: I'm stupid, I left `--quiet`

**Comment by greg0ire** [comment]
It works and I have proof! https://github.com/greg0ire/phpstan-ga/runs/537967867?check_suite_focus=true

Peek 2020-03-26 23-54

**Review by Nyholm — APPROVED** [review]
The group syntax is correct. 

The only thing I have not done before is to use a space in the group name.

**Review by phansys — APPROVED** [review]

**Comment by greg0ire** [comment]
@OskarStark please merge

File: entrypoint.sh [hunk 1]
```diff
@@ -5,4 +5,7 @@ if [ "$REQUIRE_DEV" = "true" ]; then
     NO_DEV=""
 fi
 
-sh -c "composer install --no-scripts --no-progress --quiet $NO_DEV && /composer/vendor/bin/phpstan $*"
+echo "::group::composer install"
+composer install --no-scripts --no-progress $NO_DEV
```
Comment (OskarStark) [inline entrypoint.sh]: Can we please use the exact command as group name ?

**Review by OskarStark — APPROVED** [review]

**Comment by OskarStark** [comment]
Thank you

**Pull request merged**